azuredevops/internal/service/serviceendpoint/commons.go

```diff
@@ -362,6 +371,38 @@ func dataSourceGenBaseSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		"service_endpoint_project_references": {
```

Comment:
Since this affects all those service endpoint data sources, could you please also reflect that in their documents respectively?